src/goPackages.ts

```diff
@@ -75,19 +81,23 @@ function gopkgs(workDir?: string): Promise<Map<string, string>> {
 					}
 					const index = pkgPath.lastIndexOf('/');
 					const pkgName = index === -1 ? pkgPath : pkgPath.substr(index + 1);
-					pkgs.set(pkgPath, pkgName);
+					pkgs.set(pkgPath, {
+						name: pkgName,
+						isStd: goroot === null ? false : pkgPath.startsWith(goroot)
```

Comment:
`pkgPath` is not the absolute path. It is the import path. So `pkgPath.startsWith(goroot)` here for the old gopkgs tool will always return false.

So, we might as well set this to `false` directly.

I'll soon retire this part of the code as we don't expect anyone to be using the old `gopkgs` anymore

Reply:
Or another option is to use this very crude logic I currently use for the auto completion widget

```suggestion
						isStd: !pkgPath.includes('.')
```